app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/launcher/LauncherPresenter.kt

```diff
@@ -0,0 +1,25 @@
+package org.mifos.mobile.cn.ui.mifos.launcher
+
+import android.content.Context
+import org.mifos.mobile.cn.injection.ApplicationContext
+import org.mifos.mobile.cn.ui.base.BasePresenter
+import javax.inject.Inject
+
+/**
+ * @author Manish Kumar
+ * On 31/05/18.
+ */
+
+class LauncherPresenter @Inject constructor(@ApplicationContext context: Context)
```

Comment:
remove this file we don't have any logic to write for now if you would have in future we will add it.